src/eos-crash-metrics.c

```diff
@@ -41,6 +68,22 @@ report_crash (const char *binary,
   g_variant_dict_insert_value (&dict, "ostree_commit", g_variant_new_string (ostree_commit));
   g_variant_dict_insert_value (&dict, "ostree_url", g_variant_new_string (ostree_url));
 
+  if (info->app && info->runtime)
+    {
+      g_variant_dict_insert_value (&dict, "app_name",
+                                   g_variant_new_string (flatpak_ref_get_name (FLATPAK_REF (info->app))));
+      g_variant_dict_insert_value (&dict, "app_arch",
+                                   g_variant_new_string (flatpak_ref_get_arch (FLATPAK_REF (info->app))));
+      g_variant_dict_insert_value (&dict, "app_branch",
+                                   g_variant_new_string (flatpak_ref_get_branch (FLATPAK_REF (info->app))));
+      g_variant_dict_insert_value (&dict, "app_commit",
+                                   g_variant_new_string (flatpak_ref_get_commit (FLATPAK_REF (info->app))));
+      g_variant_dict_insert_value (&dict, "runtime_name",
+                                   g_variant_new_string (flatpak_ref_get_name (FLATPAK_REF (info->runtime))));
+      g_variant_dict_insert_value (&dict, "runtime_commit",
```

Comment:
Instead of inserting a bunch of keys like this, it might be better to insert two object-valued parameters, like this:

    "app": {
        "name": ...
    },
    "runtime": {
        "name": ...
    }